### DISCUSSION TITLE
Welcome to Python-For-Beginners Discussions!

### DISCUSSION THREAD

**Discussion by Ctoic** [Ctoic/Python-For-Beginners#2] [Announcements]
## 👋 Welcome Geeks!
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Charli-drh** [comment]
Hello, my name is Charli. I am a university student, and I just started my journey. I am interested in learning python because it is a popular language and it can become a very powerful tool. I have been looking through GitHub and found some interesting projects with videos that explain a lot. Are there any recommendations for somebody like me who is new?

**Reply by Ctoic** [reply]
I would highly recommend the MIT course which is available on EDx.  Whichever course you are following the key is to practice more. For practice go to hacker-rank or Codewars where you will find many problems. The more you will solve the problems the better you will understand programming.
